File: CS/PrintingExample/ReceptionDeskContext.cs
namespace PrintingExample {
    using System;
    using System.ComponentModel.DataAnnotations;
    using System.Data.Entity;

    public class ReceptionDeskContext: DbContext {
        // Your context has been configured to use a 'ReceptionDeskContext' connection string from your application's 
        // configuration file (App.config or Web.config). By default, this connection string targets the 
        // 'EntityFrameworkExample.ReceptionDeskContext' database on your LocalDb instance. 
        // 
        // If you wish to target a different database and/or database provider, modify the 'ReceptionDeskContext' 
        // connection string in the application configuration file.
        public ReceptionDeskContext()
            : base("name=ReceptionDeskContext") {
            Database.SetInitializer(new ReceptionDeskDBInitializer());
        }

        // Add a DbSet for each entity type that you want to include in your model. For more information 
        // on configuring and using a Code First model, see http://go.microsoft.com/fwlink/?LinkId=390109.

        public virtual DbSet<MedicalAppointment> MedicalAppointments { get; set; }
        public virtual DbSet<Doctor> Doctors { get; set; }
    }

    public class Doctor {
        public Doctor() { }
        public Doctor(int id, string name) {
            Id = Id;
            Name = name;
        }
        [Key()]
        public  int Id { get; set; }
        public  string Name { get; set; }
    }

    public class MedicalAppointment {
        protected MedicalAppointment() { }
        [Key()]
        public int Id { get; set; }
        public bool AllDay { get; set; }
        [Required]
        public DateTime StartTime { get; set; }
        [Required]
        public DateTime EndTime { get; set; }
        public string PatientName { get; set; }
        public string Notes { get; set; }
        public string Subject { get; set; }
        public int PaymentStateId { get; set; }
        public int IssueId { get; set; }
        public int Type { get; set; }
        public string Location { get; set; }
        public string RecurrenceInfo { get; set; }
        public string ReminderInfo { get; set; }
        public int? DoctorId { get; set; }
        public string InsuranceNumber { get; set; }
        public bool FirstVisit { get; set; }
    }
}

File: VB/PrintingExample/ReceptionDeskContext.vb
Imports System.ComponentModel.DataAnnotations
Imports System.Data.Entity

Namespace PrintingExample

    Public Class ReceptionDeskContext
        Inherits DbContext

        ' Your context has been configured to use a 'ReceptionDeskContext' connection string from your application's 
        ' configuration file (App.config or Web.config). By default, this connection string targets the 
        ' 'EntityFrameworkExample.ReceptionDeskContext' database on your LocalDb instance. 
        ' 
        ' If you wish to target a different database and/or database provider, modify the 'ReceptionDeskContext' 
        ' connection string in the application configuration file.
        Public Sub New()
            MyBase.New("name=ReceptionDeskContext")
            Call Database.SetInitializer(New ReceptionDeskDBInitializer())
        End Sub

        ' Add a DbSet for each entity type that you want to include in your model. For more information 
        ' on configuring and using a Code First model, see http://go.microsoft.com/fwlink/?LinkId=390109.
        Public Overridable Property MedicalAppointments As DbSet(Of MedicalAppointment)

        Public Overridable Property Doctors As DbSet(Of Doctor)
    End Class

    Public Class Doctor

        Public Sub New()
        End Sub

        Public Sub New(ByVal id As Integer, ByVal name As String)
            Me.Id = Me.Id
            Me.Name = name
        End Sub

        <Key()>
        Public Property Id As Integer

        Public Property Name As String
    End Class

    Public Class MedicalAppointment

        Protected Sub New()
        End Sub

        <Key()>
        Public Property Id As Integer

        Public Property AllDay As Boolean

        <Required>
        Public Property StartTime As Date

        <Required>
        Public Property EndTime As Date

        Public Property PatientName As String

        Public Property Notes As String

        Public Property Subject As String

        Public Property PaymentStateId As Integer

        Public Property IssueId As Integer

        Public Property Type As Integer

        Public Property Location As String

        Public Property RecurrenceInfo As String

        Public Property ReminderInfo As String

        Public Property DoctorId As Integer?

        Public Property InsuranceNumber As String

        Public Property FirstVisit As Boolean
    End Class
End Namespace
